client/src/components/Navbar/Navbar.js

```diff
@@ -49,58 +51,74 @@ function Navbar() {
   return (
 		<>
 			<nav className="navbar">
-				<Link to="/" className="navbar-logo" onClick={closeMobileMenu}>
+				<NavLink to="/home" className="navbar-logo" onClick={closeMobileMenu}>
 					<img
 						src="https://i.postimg.cc/Z55sJJwz/Logo.jpg"
```

Comment:
please make this image local placed to <root>/client/src/images/
and then import it as suggested [here](https://create-react-app.dev/docs/adding-images-fonts-and-files/).
refer to Meg's sections.